### PR TITLE
Correct table header for PCE and non PCE scenarios

### PR DESCRIPTION
**Pull request by yadavsachin600**
It looks like table headers are swapped for PCE and Non-PCE option.  Things described in the third column more aligned with the scenario when using PCE. Let me know if my understanding is not correct.

### CODE REVIEW

**Comment by paigehargrave** [comment]
@sacyadav - Hi Sachin, thank you for the feedback! This is going through some restructuring, and I will have the tech reviewers verify this. Stay tuned. :)

File: modules/ROOT/pages/deployment-options.adoc [hunk 1]
```diff
@@ -3,147 +3,133 @@ ifndef::env-site,env-github[]
...
-Anypoint Platform functionality can be classified into two main categories:
+* US cloud
++
+The complete Anypoint Platform. All data is hosted within the US.
+* EU cloud
++
+Includes all US cloud components, but differs from the US cloud in that all data is hosted within the EU. Refer to xref:eu-control-plane::index.adoc[About the EU Control Plane] for additional information.
+* MuleSoft Government Cloud
++
+FedRAMP moderate authorized version of US cloud. Refer to xref:gov-cloud::index.adoc[About Government Cloud] for additional information.
```
Comment (kbostrom) [inline modules/ROOT/pages/deployment-options.adoc]: Will your audience know what "moderate authorized version" means?

File: modules/ROOT/pages/deployment-options.adoc [hunk 2]
```diff
@@ -3,147 +3,133 @@ ifndef::env-site,env-github[]
...
+Your organization's needs determine where Anypoint Platform functionality is deployed and who manages it. Anypoint Platform functionality can be classified into two main categories:
+
+* Control plane
 + 
-The Anypoint Platform runtime plane is where applications are deployed, and also where Mule runtime engine (Mule)
-and other application-related services, such as Anypoint Connectors, Anypoint Object Store, and messaging, run. 
+Anypoint Platform control plane functionality includes the design, engagement, and management capabilities of Anypoint Platform, such as Anypoint Design Center, Anypoint Management Center, and Anypoint Exchange.
+* Runtime plane
++
+Anypoint Platform runtime plane functionality includes the runtime engine (Mule) and runtime services, such as Anypoint Connectors, Anypoint Object Store, and messaging that allow you to scale the power of Mule and Anypoint Platform, and is where applications are deployed.
```
Comment (kbostrom) [inline modules/ROOT/pages/deployment-options.adoc]:
```suggestion
Anypoint Platform runtime plane is where applications are deployed. Its functionality includes the runtime engine (Mule) and runtime services (such as Anypoint Connectors, Anypoint Object Store, and messaging) that enable you to scale the power of Mule and Anypoint Platform.
```
Adding the parens limits the awkwardness of verb agreement with "enable."

File: modules/ROOT/pages/deployment-options.adoc [hunk 3]
```diff
@@ -3,147 +3,133 @@ ifndef::env-site,env-github[]
...
 
-The following table lists Anypoint Platform control and runtime plane components: 
+Anypoint Platform control plane and runtime plane functionality can be deployed on-premises, in the cloud, or in a hybrid environment in the following configurations:
```
Comment (kbostrom) [inline modules/ROOT/pages/deployment-options.adoc:31]: active voice?
```suggestion
You can deploy Anypoint Platform control plane and runtime plane functionality on-premises, in the cloud, or in a hybrid environment:
```
However, note that this intro list does not mimic the order in which the listed items are described, and it should.

File: modules/ROOT/pages/deployment-options.adoc [hunk 4]
```diff
@@ -3,147 +3,133 @@ ifndef::env-site,env-github[]
...
-* Anypoint MQ
-* Anypoint Object Store
-* Connectors
-|===
+* CloudHub
++
+MuleSoft hosts and manages public cloud infrastructure for both control and runtime planes. 
+* Hybrid
++
+MuleSoft hosts and manages public cloud infrastructure for the control plane, with the runtime plane deployed in a hybrid configuration: both Anypoint Runtime Fabric and on CloudHub.
```
Comment (kbostrom) [inline modules/ROOT/pages/deployment-options.adoc]:
```suggestion
MuleSoft hosts and manages public cloud infrastructure for the control plane, with the runtime plane deployed in a hybrid configuration: both in Anypoint Runtime Fabric and on CloudHub.
```

**Review by kbostrom — APPROVED** [review]
Here you go!

**Pull request merged**